Resources/Textures/Objects/Fun/toys.rsi/meta.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, orb, orb-inhand-left and orb-inhand-right created by Pancake",
+  "copyright": "Taken from tgstation at https://github.com/tgstation/tgstation/commit/e1142f20f5e4661cb6845cfcf2dd69f864d67432, orb, orb-inhand-left and orb-inhand-right created by Pancake, among-base and among-visor used with respect to the Among Us IP policy at https://www.innersloth.com/fan-creation-policy/",
```

Comment:
This is wrong, check your ping in shiptest for the correct copyright for the pequeno, and make a new RSI.